lib/home_page/exit_alert.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:flutter/material.dart';
+
+import '../constants/strings.dart' as strings;
+
+class ExitAlert extends StatelessWidget {
+  const ExitAlert({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return AlertDialog(
```

Comment:
Super minor improvement: in the last version of Flutter a new constructor was introduced: `AlertDialog.adaptive(...)`, which adapts to the platform you're deploying to!
You could update that, or leave it as it's right now (and resolve this conversation as well)